torch/jit/quantized.py

```diff
@@ -485,4 +561,6 @@ def quantize_rnn_modules(module):
         setattr(module, name, mod)
     if isinstance(mod, torch.nn.LSTM):
         return QuantizedLSTM(mod)
+    if isinstance(mod, torch.nn.GRU):
+        return QuantizedGRU(mod)
```

Comment:
Line 562-565: The variable `mod` is not defined in this scope. Did you mean `module`?
(the same applies to Lines 546-547)